tests/js/spec/components/events/eventCauseEmpty.spec.jsx

```diff
@@ -54,9 +57,31 @@ describe('EventCauseEmpty', function () {
     });
   });
 
+  /**
+   * Want to alternate between showing the configure suspect commits prompt and
+   * the show configure distributed tracing prompt.
+   */
+  it('doesnt render when event id starts with even char', async function () {
+    const newEvent = {
+      ...event,
+      id: '2',
+      eventID: '21098765432109876543210987654321',
```

Comment:
minor one, but could we add a test where its all letters? I could see someone missing that the parseInt's radix is 16